### DISCUSSION TITLE
test-coverage-report-action fails after updating Jest to latest version

### DISCUSSION THREAD

**Discussion by ethanryan** [ArtiomTr/jest-coverage-report-action#301] [Q&A]
All tests are passing locally, issue is only with github workflow. 

Any ideas or suggestions on how to handle this would be appreciated, thanks!

**Comment by ArtiomTr** [comment]
Hello @ethanryan :wave:,

Could you please send a screenshot of the workflow console? There are many possible reasons why action is failing.
